<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DAAligmentPositionEditWidget</class>
 <widget class="QWidget" name="DAAligmentPositionEditWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>92</width>
    <height>82</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>80</width>
    <height>70</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="leftMargin">
    <number>3</number>
   </property>
   <property name="topMargin">
    <number>3</number>
   </property>
   <property name="rightMargin">
    <number>3</number>
   </property>
   <property name="bottomMargin">
    <number>3</number>
   </property>
   <property name="spacing">
    <number>3</number>
   </property>
   <item row="0" column="2">
    <spacer name="verticalSpacer_2">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>0</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="1">
    <widget class="QToolButton" name="toolButtonTopLeft">
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="icon.qrc">
       <normaloff>:/DACommonWidgets/icon/topleft.svg</normaloff>:/DACommonWidgets/icon/topleft.svg</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>18</width>
       <height>18</height>
      </size>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <property name="autoRaise">
      <bool>true</bool>
     </property>
     <attribute name="buttonGroup">
      <string notr="true">buttonGroup</string>
     </attribute>
    </widget>
   </item>
   <item row="1" column="2">
    <widget class="QToolButton" name="toolButtonTop">
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="icon.qrc">
       <normaloff>:/DACommonWidgets/icon/top.svg</normaloff>:/DACommonWidgets/icon/top.svg</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>18</width>
       <height>18</height>
      </size>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <property name="autoRaise">
      <bool>true</bool>
     </property>
     <attribute name="buttonGroup">
      <string notr="true">buttonGroup</string>
     </attribute>
    </widget>
   </item>
   <item row="1" column="3">
    <widget class="QToolButton" name="toolButtonTopRight">
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="icon.qrc">
       <normaloff>:/DACommonWidgets/icon/topRight.svg</normaloff>:/DACommonWidgets/icon/topRight.svg</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>18</width>
       <height>18</height>
      </size>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <property name="autoRaise">
      <bool>true</bool>
     </property>
     <attribute name="buttonGroup">
      <string notr="true">buttonGroup</string>
     </attribute>
    </widget>
   </item>
   <item row="2" column="0">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>0</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="2" column="1">
    <widget class="QToolButton" name="toolButtonLeft">
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="icon.qrc">
       <normaloff>:/DACommonWidgets/icon/left.svg</normaloff>:/DACommonWidgets/icon/left.svg</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>18</width>
       <height>18</height>
      </size>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <property name="autoRaise">
      <bool>true</bool>
     </property>
     <attribute name="buttonGroup">
      <string notr="true">buttonGroup</string>
     </attribute>
    </widget>
   </item>
   <item row="2" column="2">
    <widget class="QToolButton" name="toolButtonCenter">
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="icon.qrc">
       <normaloff>:/DACommonWidgets/icon/center.svg</normaloff>:/DACommonWidgets/icon/center.svg</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>18</width>
       <height>18</height>
      </size>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <property name="autoRaise">
      <bool>true</bool>
     </property>
     <attribute name="buttonGroup">
      <string notr="true">buttonGroup</string>
     </attribute>
    </widget>
   </item>
   <item row="2" column="3">
    <widget class="QToolButton" name="toolButtonRight">
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="icon.qrc">
       <normaloff>:/DACommonWidgets/icon/right.svg</normaloff>:/DACommonWidgets/icon/right.svg</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>18</width>
       <height>18</height>
      </size>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <property name="autoRaise">
      <bool>true</bool>
     </property>
     <attribute name="buttonGroup">
      <string notr="true">buttonGroup</string>
     </attribute>
    </widget>
   </item>
   <item row="2" column="4">
    <spacer name="horizontalSpacer_2">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>0</width>
       <height>19</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="3" column="1">
    <widget class="QToolButton" name="toolButtonBottomLeft">
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="icon.qrc">
       <normaloff>:/DACommonWidgets/icon/bottomLeft.svg</normaloff>:/DACommonWidgets/icon/bottomLeft.svg</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>18</width>
       <height>18</height>
      </size>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <property name="autoRaise">
      <bool>true</bool>
     </property>
     <attribute name="buttonGroup">
      <string notr="true">buttonGroup</string>
     </attribute>
    </widget>
   </item>
   <item row="3" column="2">
    <widget class="QToolButton" name="toolButtonBottom">
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="icon.qrc">
       <normaloff>:/DACommonWidgets/icon/bottom.svg</normaloff>:/DACommonWidgets/icon/bottom.svg</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>18</width>
       <height>18</height>
      </size>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <property name="autoRaise">
      <bool>true</bool>
     </property>
     <attribute name="buttonGroup">
      <string notr="true">buttonGroup</string>
     </attribute>
    </widget>
   </item>
   <item row="3" column="3">
    <widget class="QToolButton" name="toolButtonBottomRight">
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="icon.qrc">
       <normaloff>:/DACommonWidgets/icon/bottomRight.svg</normaloff>:/DACommonWidgets/icon/bottomRight.svg</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>18</width>
       <height>18</height>
      </size>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <property name="autoRaise">
      <bool>true</bool>
     </property>
     <attribute name="buttonGroup">
      <string notr="true">buttonGroup</string>
     </attribute>
    </widget>
   </item>
   <item row="4" column="2">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>1</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="icon.qrc"/>
 </resources>
 <connections/>
 <buttongroups>
  <buttongroup name="buttonGroup"/>
 </buttongroups>
</ui>
